.github/workflows/bump-openapi.yml

```diff
@@ -0,0 +1,31 @@
+name: bump-openapi
+on:
+  workflow_dispatch:
+    inputs:
+      sentry-api-schema-sha:
+        description: 'The "sentry-api-schema" SHA to run against'
+        required: false
+  push:
+    branches:
+      - master
```

Comment:
Should this run on pushes to master?

Reply:
hmm my mistake, it should only run on the workflow dispatch trigger